build-ext

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env php
+<?php
+
+const CONFIG_M4_PATH = './ext/config.m4';
+const DOCKER_MATCH = 'PHP_ADD_BUILD_DIR([$ext_builddir/tensor/])';
+const DOCKER_PATCH = DOCKER_MATCH . "\n" . <<<'EOT'
+	PHP_ADD_BUILD_DIR([$ext_builddir/tensor/decompositions/])
+	PHP_ADD_BUILD_DIR([$ext_builddir/tensor/exceptions/])
+	PHP_ADD_BUILD_DIR([$ext_builddir/tensor/reductions/])
+
+	AC_CHECK_FUNC(backtrace_symbols, have_backtrace_symbols=yes, have_backtrace_symbols=no)
+	if test $have_backtrace_symbols = no; then
+		LDFLAGS="${LDFLAGS:-} -lexecinfo"
+	fi
+
+EOT
+;
+
+chdir(__DIR__);
+
+passthru('zephir generate', $rc);
```

Comment:
Will this only generate the C source? What happens if we want to compile the whole extension from Zephir and C source like we currently do with `$ composer compile`?

Reply:
`build-ext` is meant to be a wrapper of `'zephir generate` only.